packages/x-components/src/components/animations/animate-width.vue

```diff
@@ -15,7 +15,7 @@
    * @public
    */
   export default defineComponent({
-    inheritAttrs: false
```

Comment:
Not sure the side effects of this change but in order to keep the functionality as it is and avoid unexpected behaviours I'd keep it as it was.

Reply:
Restored